### DISCUSSION TITLE
Adding Minecraft World Distrubution

### DISCUSSION THREAD

**Discussion by Bowswa** [modrinth/.github#108] [Feature requests]
Since Curseforge has become less than awesome whilst playing musical chairs with the chairs being companies to own it, Modrinth has been becoming the replacement. But this platform is still missing one thing, Maps. 

I recently made a breakthrough on my map, which has been based on curseforge for a while. I have no functioning release, as i have been stuck on a single bug for a while. Now that i have got it, I would like to be able to move over to Modrinth, but there is no section for maps.

**Comment by Minenash** [comment]
Duplicate of #99
